deploy/csi-operator.yaml

```diff
@@ -923,15 +923,15 @@ roleRef:
 apiVersion: scheduling.k8s.io/v1
 kind: PriorityClass
 metadata:
-  name: openebs-csi-controller-critical
+  name: openebs-cstor-csi-controller-critical
```

Comment:
Can we update the same here: https://github.com/openebs/cstor-operators/blob/master/deploy/csi-operator.yaml#L926